README.md

```diff
@@ -41,24 +41,22 @@ Profiling in a production environment is not recommend.
 
 ### Drupal 8+
 
-The `xhgui/examples` contains files which can be used to quick-start a Drupal installation.
+The `xhgui/examples` contains files to quick-start a Drupal installation configuration.
 
 - Copy the files from `xhgui/examples` to the sites's `web/sites/default` folder.
 
-- Add the following line to `web/sites/default/settings.php` to include the collector.
+- Add the following line to `web/sites/default/settings.local.php` to include the collector.
 
    ```php
    require_once __DIR__ . '/xhgui.collector.php';
    ```
 
-You can update `web/sites/default/settings.ddev.php` instead, however this file will be overwritten when DDEV restarts unless you remove `#ddev-generated` from the top of the file.
+- Run `ddev xhprof` to start profiling.
 
-- Comment out the above line to disable profiling.
-
-### Wordpress
+### WordPress
 
-Download latest version of `perftools/php-profiler` (this has been validated with the current latest release, 0.18.0).
-If you use [bedrock](https://roots.io/bedrock/), just use the composer command from the previous section.
+Download latest version of `perftools/php-profiler` (validated with the current latest release, 0.18.0).
```

Comment:
This *seemed* to have been covered above, but of course not all WP are composer installations. But maybe that means that the wording above saying to composer-install it needs to be fiddled with.

Reply:
I moved the `perftools/php-profiler` installation into each framework.
It feels like php-profiler is tied to the framework and its composer restrictions. 
I'm OK with centralizing it if you feel strongly.